Appendix/standard deviation (#55)
MrLongbottom/P10-Article: sections/appendix/standard_dev.tex

\subsection{\gls{lda} standard deviation}\label{sec:std_dev}
Since the \gls{lda} model uses a generative process, we see how confident the standard model is, in the resulting topic coherence.
We have run 10 models using the hyper parameters $K=90$, $\alpha=0.01$ and $\beta=0.1$.

> Measuring the confidence of our Gibbs sampler, can be calculated by the standard deviation by using the results from the 10 models' topic coherence.

Author: ```suggestion
Measuring the confidence of our Gibbs sampler can be calculated by the standard deviation by using the results from the 10 models' topic coherence.

```

> \subsection{\gls{lda} standard deviation}\label{sec:std_dev}

Reviewer: ```suggestion
\subsection{\Gls{lda} standard deviation}\label{sec:std_dev}
```

> Since the \gls{lda} model uses a generative process, we see how confident the standard model is in the resulting topic coherence.

Reviewer 2: what is calculating confidence / standard deviation good for?

> Since the \gls{lda} model uses a generative process, we see how confident the standard model is, in the resulting topic coherence.

Reviewer: ```suggestion
Since the \gls{lda} model uses a generative process, we see how confident the standard model is in the resulting topic coherence.
```

> We have run 10 models using the hyper parameters $K=90$, $\alpha=0.01$ and $\beta=0.1$.

Reviewer: ```suggestion
We have run 10 models using the hyperparameters $K=90$, $\alpha=0.01$, and $\beta=0.1$.
```

> Measuring the confidence of our Gibbs sampler can be calculated by the standard deviation by using the results from the 10 models' topic coherence.

Reviewer: ```suggestion
The confidence of our Gibbs sampler can be calculated by the standard deviation of the results from the 10 models' topic coherence scores.
```

We calculate the standard deviation by:
\begin{equation}

> 	SD = \sqrt{\frac{\sum_{i=1}^{n}(x_i-\bar{x})^2}{n-1}}

Reviewer: Jeg kan se at formlen er en smule anderledes afhængig af om man ser sin data som en fuld population eller et sample af en population. Jeg ville nok tage vores 10 resultater som en population, og ændre formlen til det, men hvis i har et argument for at tænke på det som et sample kan vi også gå med det.

Author: jeg var lidt i tvilv om hvilken en det var :)

Reviewer: Jeg er personligt heller ikke sikker på hvad er rigtigt, for jeg kunne ikke finde et tydeligt svar på hvad gælder for os. Hvis vi bruger population versionen skal ligningen lige ændres, ellers er jeg tilfreds.

\end{equation}

> where $\mu = 0.535$, which is the mean of the 10 models.

Reviewer: ```suggestion
where $n$ is the number of results, $x_i$ is the current result, and $\mu = 0.535$ is the mean of the 10 models.
```

Reviewer: Hvis formlen forbliver på sample form, skal \mu ændres til \bar{x}

> The standard deviation for our \gls{lda} model is $0.0104$, which means that our model does not deviate much from the mean.

Reviewer: ```suggestion
The standard deviation for our \gls{lda} model is $0.0104$, which means that our model does not deviate much from the mean, even though the models have different starting configurations.
```

> The standard deviation for our \gls{lda} model is $0.0104$, which means that our model does not deviate much from the mean, even though the models have different starting configurations.

Reviewer 2: hvis det skal kunne bruges til at vise at den farvede table er relevant skal vi måske også vise en farvet table her?
